roles/lib_openshift/src/class/oc_route.py

```diff
@@ -55,7 +55,13 @@ def create(self):
 
     def update(self):
         '''update the object'''
-        # need to update the tls information and the service name
+        # when the host attribute is being updated, we need to delete and recreate
```

Comment:
@detiber, I didn't actually pass the `--force` in my example.  Thanks for doing that.

I already have  _replace_content take a force flag so I'm updating to call `--force`.

Good find.